docs/requirements.txt

```diff
@@ -1,8 +1,12 @@
+docutils<0.18
 ipython
+m2r2
 netCDF4
 numpydoc
 pandas
```

Comment:
Packaging?

Reply:
we do reference `packaging` in the docs via pymod, so it might be needed here.

Reply:
Added